### PR TITLE
fix: fixed depends_on for sec and cm

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Fixed explicit depends on for secrets and config maps.
[Git Issue](https://github.com/terraform-ibm-modules/terraform-ibm-code-engine/issues/113)

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by akocbek — DISMISSED** [review]

File: main.tf [hunk 1]
```diff
@@ -120,7 +121,7 @@ module "build" {
 # Code Engine Domain Mapping
 ##############################################################################
 module "domain_mapping" {
```
Comment (akocbek) [inline main.tf:123]: if I am not wrong domain mapping should still depend on `secret` and not the `app`. As we can see the object has reference to `domain_mappings` which has `tls_secret` and project id

Comment (Ak-sky) [reply main.tf:123]: Done.

Comment (Ak-sky) [reply main.tf:123]: I have added both secret and app.

**Review by akocbek — APPROVED** [review]

**Pull request merged**